runtime/near-vm-runner/src/logic/logic.rs

```diff
@@ -1777,11 +1778,15 @@ impl<'a> VMLogic<'a> {
             sir,
             self.config.implicit_account_creation,
             receiver_id.get_account_type(),
+            // TODO(eth-implicit) What protocol version to use?
+            PROTOCOL_VERSION,
         );
         let exec_fee = transfer_exec_fee(
             self.fees_config,
             self.config.implicit_account_creation,
             receiver_id.get_account_type(),
+            // TODO(eth-implicit) What protocol version to use?
+            PROTOCOL_VERSION,
```

Comment:
Looking at the rest of the runtime code, I think the pattern to follow is to create a new field in the runtime config (similar to the `implicit_account_creation` flag for example), then conditionally set that field depending on the protocol version in the `RuntimeConfigStore` https://github.com/near/nearcore/blob/master/core/primitives/src/runtime/config_store.rs

Using the pattern the runtime does not need to know anything about the protocol version itself.